Automattic/vip-go-mu-plugins
# Async Cmd Scheduler: trim whitespaces from Slack audit messages, adjust message format to ensure there's no misrendered markdown

## Description

This is a minor formatting issue fix. Due to a bug in the code whitespaces coming from PHP formatting weren't appropriately stripped, resulting in weird-looking messages.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Apply the PR.
2. Add `echo $formatted_message` below #L161
3. in `wp shell` run `\Automattic\VIP\Commands\Async_Scheduler_Command::runner( 'option what home' );`
4. Observe that the formatted message doesn't have any leading white spaces.
